File: Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/IInterstitialEventListener.java
package com.chartboost.heliumsdk.unity;

public interface IInterstitialEventListener {
    void DidLoadInterstitial(String placementName, int errorCode, String errorDescription);
    void DidClickInterstitial(String placementName, int errorCode, String errorDescription);
    void DidCloseInterstitial(String placementName, int errorCode, String errorDescription);
    void DidShowInterstitial(String placementName, int errorCode, String errorDescription);
    void DidWinBidInterstitial(String placementName, String partnerPlacementName,  String auctionId, double price, String seat);
}

File: com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/src/main/java/com/chartboost/mediation/unity/IInterstitialEventListener.kt
package com.chartboost.mediation.unity

/**
 * Interface for sending real-time Interstitial events
 */
@Deprecated("IInterstitialEventListener has been deprecated, utilize the new Fullscreen API instead")
interface IInterstitialEventListener {
    fun DidLoadInterstitial(placementName: String, loadId: String, auctionId: String, partnerId: String, price: Double, lineItemId: String, error: String)
    fun DidShowInterstitial(placementName: String, error: String)
    fun DidCloseInterstitial(placementName: String, error: String)
    fun DidClickInterstitial(placementName: String)
    fun DidRecordImpression(placementName: String)
    fun DidReceiveReward(placementName: String)
}
